jb/itch5/compute_inside.hpp

```diff
@@ -1,200 +1,48 @@
 #ifndef jb_itch5_compute_inside_hpp
 #define jb_itch5_compute_inside_hpp
 
-#include <jb/itch5/order_book.hpp>
-#include <jb/itch5/add_order_message.hpp>
-#include <jb/itch5/add_order_mpid_message.hpp>
-#include <jb/itch5/order_cancel_message.hpp>
-#include <jb/itch5/order_delete_message.hpp>
-#include <jb/itch5/order_executed_message.hpp>
-#include <jb/itch5/order_executed_price_message.hpp>
-#include <jb/itch5/order_replace_message.hpp>
-#include <jb/itch5/stock_directory_message.hpp>
-#include <jb/itch5/unknown_message.hpp>
-
-#include <boost/functional/hash.hpp>
-#include <chrono>
-#include <functional>
-#include <unordered_map>
+#include <jb/itch5/compute_base.hpp>
 
 namespace jb {
 namespace itch5 {
 
-/**
- * An implementation of jb::message_handler_concept to compute the inside.
- *
- * Keep a collection of all the order books, and forward the right
- * updates to them as it handles the different message types in
- * ITCH-5.0.
+/** 
+ * An implementation derivated fomr compute_base to callback the inside.
```

Comment:
Please describe what the class does, not how it is implemented, e.g.

    /**
     * A book builder that notifies only on changes to the inside.